.github/workflows/build-n-publish.yml

```
@@ -20,10 +20,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1.4.3
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          java-version: 8
```

```suggestion
          java-version: '8'
```
Don't this need to be in `''` ?